metricbeat/metricbeat.full.yml

```diff
@@ -1,470 +1,28 @@
-########################## Metricbeat Configuration ###########################
+###################### Metricbeat Configuration Example #######################
```

Comment:
What happens to the full config? As now each module is separate, should the config there directly contain the full config and only have 1 version?